doc/v2-manifesto.md

```diff
@@ -0,0 +1,27 @@
+## What sucks about Squirrel v1's Design™
+
+* Squirrel.Windows and Squirrel.Mac are completely different
+  - This confuses the living fuck out of people
+
+* Squirrel.Mac is missing lots of features
+  - Delta updates
+  - Updates via static HTTPS (i.e. S3)
+  - Rollout updates 
+
```

Comment:
Hooks are generally a Windows-only feature since you don't even get a chance to do anything on uninstall on macOS, but it's important on Windows (since you have to remove stuff on uninstall). We should probably remove them all except for uninstall tbh.